src/optim_problems/run_opt.jl

```diff
@@ -1,27 +1,26 @@
 """
-    run_opt(ts_data::ClustData,opt_data::OptDataCEP,opt_config::Dict{String,Any};solver::Any=CbcSolver())
+    run_opt(ts_data::ClustData,opt_data::OptDataCEP,opt_config::Dict{String,Any};optimizer::DataType)
 organizing the actual setup and run of the CEP-Problem
 """
 function run_opt(ts_data::ClustData,
                     opt_data::OptDataCEP,
-                    opt_config::Dict{String,Any};
-                    solver::Any=CbcSolver(),
-                    k_ids::Array{Int64,1}=Array{Int64,1}()
+                    opt_config::Dict{String,Any},
+                    optimizer::DataType
                     )
   #Check the consistency of the data provided
   check_opt_data_cep(opt_data)
-  cep=setup_opt_cep_basic(ts_data, opt_data, opt_config, solver; k_ids=k_ids)
+  cep=setup_opt_cep_basic(ts_data, opt_data, opt_config, optimizer, opt_config["optimizer_config"])
   setup_opt_cep_basic_variables!(cep, ts_data, opt_data)
   if opt_config["lost_load_cost"]["el"]!=Inf
     setup_opt_cep_lost_load!(cep, ts_data, opt_data)
   end
   if opt_config["lost_emission_cost"]["CO2"]!=Inf
     setup_opt_cep_lost_emission!(cep, ts_data, opt_data)
   end
-  if opt_config["storage_p"] && opt_config["storage_e"] && opt_config["seasonalstorage"]
+  if opt_config["storage_in"] && opt_config["storage_out"] && opt_config["storage_e"] && opt_config["seasonalstorage"]
```

Comment:
storage has separate technologies for `in` and `out` `p` is deprecated

Reply:
So default is p, and then e signifies energy component? Is this in the documentation?